autoextract/aio/client.py

```diff
@@ -147,6 +259,8 @@ def request_parallel_as_completed(query: Query,
                                   batch_size=1,
                                   n_conn=1,
                                   agg_stats: AggStats = None,
+                                  handle_retries=True,
+                                  skip_query_errors=False,
```

Comment:
When we settle on exact API, it'd be good to document new arguments in a docstring